### PR TITLE
Use tf12 container version

### PR DESCRIPTION
**Pull request by bgeesaman**
Modify Jenkinsfile to use the template style that reliably pulls the latest container version.

### CODE REVIEW

**Comment by bgeesaman** [comment]
test this please

**Comment by bgeesaman** [comment]
test this please

**Comment by bgeesaman** [comment]
test this please

**Comment by bgeesaman** [comment]
test this please

File: Jenkinsfile [hunk 1]
```diff
@@ -24,77 +24,73 @@ limitations under the License.
 // set up pod label and GOOGLE_APPLICATION_CREDENTIALS (for Terraform)
 def label = "k8s-infra"
 def containerName = "k8s-node"
```
Comment (chrislovecnm) [inline Jenkinsfile]: We need the new container name here. Seem to be working with that change.

**Review by chrislovecnm — DISMISSED** [review]

**Review by chrislovecnm — APPROVED** [review]

**Pull request merged**